### DISCUSSION TITLE
Timeline for GA launch?

### DISCUSSION THREAD

**Discussion by alexkirsz** [awslabs/aws-sdk-rust#102] [General]
It's only been a month since the alpha launched, but the number of services supported by the SDK is growing rapidly. With support for AWS Query, the API surface is becoming already large enough to sustain a good number of applications. With that in mind:

* Could the team share their timeline for the GA launch?
* Will the first production-ready release target feature parity with other AWS SDKs, or will it be more focused in scope?
* In the latter case, have you already decided on a minimum viable set of features?

**Comment by rcoh** [comment]
We understand that our timelines for GA impact builders who need clarity on when they can adopt the SDK for their workloads with confidence. Although we cannot share an exact timeline right now, we will share one as soon as we are able. For our GA release we are targeting feature parity with other AWS SDKs with production-ready code meeting our [official SDK maintenance policy](https://docs.aws.amazon.com/sdkref/latest/guide/maint-policy.html).

At the same time we are actively working to bring services already supported by the AWS SDK for Rust up to a production-ready quality bar.

We will continue publish progress updates via [GitHub releases](https://github.com/awslabs/aws-sdk-rust/releases) and in the README of the repo. In the meantime, please file and upvote issues for services and features you need for your application via GitHub issues (https://github.com/awslabs/aws-sdk-rust/issues); Community feedback plays a key role in guiding our prioritization.
